src/main/webapp/WEB-INF/portlet.xml

```diff
@@ -104,6 +104,14 @@
                 <value>false</value>
                 <read-only>false</read-only>
             </preference>
+            <!-- Controls what style to display in each view, based on respondr property.
+                 If respondr is used, then display normal pattern in every view.
+                 Otherwise, the appropriate mobile view theme is displayed in the layout of the portlet -->
+            <preference>
+                <name>AnnouncementsViewController.respondr</name>
+                <value>true</value>
+                <read-only>false</read-only>
+            </preference>
```

Comment:
Rather than using a portlet preference which is absolute (set for the portlet no matter which theme is used), a better solution is to either:

1. Modify the ThemeViewNameSelector to respect the theme 'respondr' and not do the mobile view in that case.  See https://github.com/Jasig/AnnouncementsPortlet/blob/master/src/main/java/org/jasig/portlet/announcements/controller/ThemeViewNameSelector.java.
2.  change the portlet to look for role 'respondr' which is set by a PAGS group (https://github.com/Jasig/uPortal/blob/uportal-4.2.1/uportal-war/src/main/data/default_entities/pags-group/Respondr_Theme_Users.pags-group.xml).

I strongly prefer the first suggested approach.